ext/sodium/sodium_pwhash.stub.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/** @generate-class-entries */
+
+/**
+ * @var string
+ * @cvalue "argon2i"
```

Comment:
you can directly assign constant values/expressions for the PHP constant, and remove this `cvalue`.